src/models/rfcProps.js

```diff
@@ -27,6 +27,7 @@ import ActionCopyNtoFN from '../components/Actions/ActionCopyNtoFN'
 import ActionToggleYear from '../components/Actions/ActionToggleYear'
 import zones from './zones'
 
+const defaultProfileState = loadState('contacts', 'defaultProfile')
```

Comment:
Could you add a little comment here to explain what we're doing, so that anyone reading this will understand what it's exactly :)